src-vue/src/views/mods/ThunderstoreModsView.vue

```diff
@@ -79,9 +80,11 @@ export default defineComponent({
             return this.mods.filter((mod: ThunderstoreMod) => {
                 // Filter with search words (only if search field isn't empty)
                 const inputMatches: boolean = this.searchValue.length === 0
-                    || (mod.name.toLowerCase().includes(this.searchValue)
-                        || mod.owner.toLowerCase().includes(this.searchValue)
-                        || mod.versions[0].description.toLowerCase().includes(this.searchValue));
+                    || (
+                        fuzzy_filter(mod.name, this.searchValue) ||
+                        fuzzy_filter(mod.owner, this.searchValue) ||
+                        mod.versions[0].description.toLowerCase().includes(this.searchValue)

```

Comment:
No fuzzy filtering on mods' descriptions? :(

Reply:
It's cause that would bring up a lot of noise/false positives.

For comparison, results with same search term as PR description but with fuzzy filter on Thunderstore description enabled:
![image](https://github.com/R2NorthstarTools/FlightCore/assets/40122905/9ed5d405-7870-48c4-a157-6719dd7c9ee4)

Reply:
Fair!